scip: 0.5.1 -> 0.6.1

kyehn opened the pull request:
Diff: https://github.com/sourcegraph/scip/compare/v0.5.1...v0.6.1

Changelog: https://github.com/sourcegraph/scip/blob/v0.6.1/CHANGELOG.md


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

kyehn committed:
scip: 0.5.1 -> 0.6.0

Diff: https://github.com/sourcegraph/scip/compare/v0.5.1...v0.6.0

Changelog: https://github.com/sourcegraph/scip/blob/v0.6.0/CHANGELOG.md

Sigmanificient commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 448358`
Commit: [`870a27dacd2ccfde1d2d1a96ef7d4f33cdff934c`](https://github.com/NixOS/nixpkgs/commit/870a27dacd2ccfde1d2d1a96ef7d4f33cdff934c) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/870a27dacd2ccfde1d2d1a96ef7d4f33cdff934c..pull/448358/head))
Merge: [`eafa1d61923d011fb728c6d8b195e9b5219e433c`](https://github.com/NixOS/nixpkgs/commit/eafa1d61923d011fb728c6d8b195e9b5219e433c)

Logs: https://github.com/Sigmanificient/nixpkgs-review-gha/actions/runs/18250816493


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>scip</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>scip</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>scip</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>scip</li>
  </ul>
</details>

kyehn committed:
scip: 0.6.0 -> 0.6.1

Diff: https://github.com/sourcegraph/scip/compare/v0.6.0...v0.6.1

Changelog: https://github.com/sourcegraph/scip/blob/refs/tags/v0.6.1/CHANGELOG.md

liberodark commented:
Hi,

Thank for your work @kyehn.
Can you remove `"-w"` is not need.

Best Regards
